### PR TITLE
Spree pr fixes: Add spree_dev_tools and gemfile 4.1 version

### PR DESCRIPTION
**Pull request by shradhaN**
Fixes according to this:

hey @poudelprakash thank you for your work, before I review this PR could you please adjust Spree versions ? At this moment we adjusting gem's to according Spree versions: 3.7, 4.0, 4.1 and master

Also, please put to this PR our testing tool: https://github.com/spree/spree_dev_tools
here you can find guide how to use it: https://guides.spreecommerce.org/developer/tutorials/upgrading_extensions.html#migrate-to-spree-dev-tools

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,10 +10,14 @@ env:
 gemfile:
   - gemfiles/spree_3_1.gemfile
   - gemfiles/spree_3_2.gemfile
+  - gemfiles/spree_3_7.gemfile
+  - gemfiles/spree_4_0.gemfile
+  - gemfiles/spree_4_1.gemfile
```
Comment (poudelprakash) [inline .travis.yml:15]: I don't see this gemfile in gemfile folder!

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,73 +1,12 @@
-# Run Coverage report
```
Comment (poudelprakash) [inline spec/spec_helper.rb:1]: can we not remove simplecov?

File: spree_slider.gemspec [hunk 3]
```diff
@@ -21,22 +21,5 @@ Gem::Specification.new do |s|
...
-  s.add_development_dependency 'mysql2'
-  s.add_development_dependency 'pg'
-  s.add_development_dependency 'pry'
-  s.add_development_dependency 'rspec-mocks'
-  s.add_development_dependency 'rspec-rails', '~> 4.0.0.beta2'
-  s.add_development_dependency 'sass-rails'
-  s.add_development_dependency 'simplecov'
-  s.add_development_dependency 'sqlite3'
-  s.add_development_dependency 'webdrivers', '~> 4.0.0'
+  s.add_development_dependency 'spree_dev_tools'
```
Comment (poudelprakash) [inline spree_slider.gemspec:24]: don't we need any of the dependencies we have here?

File: spec/spec_helper.rb (hunk 2 above)
Comment (shradhaN) [reply spec/spec_helper.rb:1]: My guess is that the `spree_dev_tools` automatically contains simplecov, therefore we do not need it. Here is a sample PR for this change https://github.com/spree/spree_gateway/pull/357/files as present in official documentation.

File: .travis.yml (hunk 1 above)
Comment (shradhaN) [reply .travis.yml:15]: spree_3_7.gemfile and spree_4_0.gemfile are already present. Just added spree_4_1.gemfile in this PR

File: spree_slider.gemspec (hunk 3 above)
Comment (shradhaN) [reply spree_slider.gemspec:24]: Please see above comment.

File: .travis.yml (hunk 1 above)
Comment (poudelprakash) [reply .travis.yml:15]: yeah..gemfiles/spree_4_1.gemfile this file is missing?

**Pull request merged**